packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Details/cards/ChartsCard.tsx

```diff
@@ -0,0 +1,160 @@
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { Chart, ChartAxis, ChartBar, ChartGroup, ChartTooltip } from '@patternfly/react-charts';
+import {
+  Card,
+  CardBody,
+  CardTitle,
+  Dropdown,
+  DropdownItem,
+  Flex,
+  FlexItem,
+  KebabToggle,
+} from '@patternfly/react-core';
+
+import { MigrationsCardProps } from './MigrationsCard';
+
+interface MigrationDataPoint {
+  dateLabel: string;
+  value: number;
+}
+
+const toStarted = (m: V1beta1Migration): string => m.status.started;
+const toFinished = (m: V1beta1Migration): string => m.status.completed;
+const hasTimestamp = (timestamp: string) => timestamp && DateTime.fromISO(timestamp).isValid;
+const toDateTime = (timestamp: string): DateTime => DateTime.fromISO(timestamp);
+const isLastWeek = (date: DateTime) => date.diffNow('days').get('days') <= 7;
+const isLastDay = (date: DateTime) => date.diffNow('hours').get('hours') <= 24;
+const toDayLabel = (date: DateTime): string => date.toFormat('LLL dd');
+const toHourLabel = (date: DateTime): string => date.toFormat('HH');
+const groupByLabel = (acc: { [key: string]: number }, label: string) => ({
+  ...acc,
+  [label]: (acc[label] || 0) + 1,
+});
+const toDataPoints = (
+  allMigrations: V1beta1Migration[],
+  toTimestamp: (m: V1beta1Migration) => string,
+  isWeekViewSelected: boolean,
+): MigrationDataPoint[] =>
+  Object.entries(
+    allMigrations
+      .map(toTimestamp)
+      .filter(hasTimestamp)
+      .map(toDateTime)
+      .filter(isWeekViewSelected ? isLastWeek : isLastDay)
+      .map(isWeekViewSelected ? toDayLabel : toHourLabel)
+      .reduce(groupByLabel, {}),
+  ).map(([dateLabel, value]) => ({ dateLabel, value }));
+
+export const ChartsCard: React.FC<MigrationsCardProps> = () => {
+  const { t } = useForkliftTranslation();
+  const [isDropdownOpened, setIsDropdownOpened] = useState(false);
+  const onToggle = () => setIsDropdownOpened(!isDropdownOpened);
+  const [isWeekViewSelected, setIsWeekViewSelected] = useState(true);
+  const [allMigrations] = useK8sWatchResource<V1beta1Migration[]>({
+    groupVersionKind: MigrationModelGroupVersionKind,
+    namespaced: true,
+    isList: true,
+  });
+  const migrations: {
+    started: MigrationDataPoint[];
+    failed: MigrationDataPoint[];
+    finished: MigrationDataPoint[];
+  } = {
+    started: toDataPoints(allMigrations, toStarted, isWeekViewSelected),
+    finished: toDataPoints(allMigrations, toFinished, isWeekViewSelected),
+    failed: toDataPoints(
+      allMigrations.filter((m) => m?.status?.conditions?.find((it) => it?.type === 'Failed')),
+      toFinished,
+      isWeekViewSelected,
+    ),
+  };
+
+  return (
+    <Card>
+      <CardTitle>{t('Last Migrations')}</CardTitle>
+      <CardBody className="forklift-status-migration">
+        <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+          <FlexItem align={{ default: 'alignRight' }}>
+            <Dropdown
```

Comment:
can you use card actions ?
https://www.patternfly.org/2023.02/components/card#header-images-and-actions